### ISSUE TITLE
spring web mvc does not generate in the documentation response body - it shows data type (custom) instead of String 

### ISSUE THREAD

**Issue by sconstro** [stoicflame/enunciate#380]
Example with enunciate-2.3 and gradle plugin:   

```java
 /**
     * @returnWrapped java.lang.String
     */
 @StatusCodes({
        @ResponseCode(code=200,condition = "ok")
    })
    public @ResponseBody String getString(){
}
```

Documentation will be generated like this:

Response Body 
media type 	                                data type
application/json;charset=UTF-8 	(custom)

**Comment by stoicflame** [comment]
What would be the expected response data type?

If the return type is String, and the media type is JSON, then presumably you're going to return some kind of JSON object in the String, right? Enunciate has no way of knowing what data type you're returning in the String, so it just says "custom".

**Comment by sconstro** [comment]
I have the same data type (custom) even if I return an object of type Person. It always appear (custom) in the documentation doesn't matter what I return

**Comment by stoicflame** [comment]
Okay, I'll take a closer look. If you could put together a simple project that demonstrates the issue that would be helpful.

**Comment by sconstro** [comment]
[enuciate-demo.zip](https://github.com/stoicflame/enunciate/files/246368/enuciate-demo.zip)

I found the issue: in the class HelloController, I added the annotation @RequestMapping(produces = "application/json;charset=UTF-8")
In this case, the "/test" will show as response body  data type (custom)
Is it any workaround?

**Comment by stoicflame** [comment]
Ah. Yes. So you have to specify somewhere that the method produces JSON; otherwise, Enunciate has no idea what datatype to link to or how to interpret the `Person` object. If you don't specify, it could be YAML or plain text or protobuf or any other kind of serialization format that Enunciate doesn't know how to document. But when you specify JSON, Enunciate knows what the `Person` object will look like and can document it.

Can you suggest any better behavior?

**Comment by stoicflame** [comment]
Fixed at e3a9e37.

**Comment by stoicflame** [comment]
[Enunciate 2.4 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.4.0).
